hack/verify-flags/exceptions.txt

```diff
@@ -5,11 +5,13 @@ cluster/aws/templates/configure-vm-aws.sh:  # We set the hostname_override to th
 cluster/aws/templates/configure-vm-aws.sh:  api_servers: '${API_SERVERS}'
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "hostname_override"
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "runtime_config"
+cluster/aws/templates/configure-vm-aws.sh:  kubelet_api_servers: '${KUBELET_APISERVER}'
```

Comment:
Travis-ci built failed previously and suggested to run `hack/verify-flags-underscore.py -e > hack/verify-flags/exceptions.txt`. I think the reason is that the new flag we're introducing here "kubelet_api_servers" is also used by these scripts, but they're not related.